src/redux/books/books.js

```diff
@@ -0,0 +1,16 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const booksReducers = (state = [], action) => {
+  switch (action.type) {
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Your books reducer still needs to implement the actual addition and removal of books. Right now it just returns the default state.

  ```
  const reducer = (state = [], action) => {
    switch (action.type) {
      case ADD_BOOK:
        return [
          action.payload,
          ...state,
        ];
      case REMOVE_BOOK:
        return state.filter((book) => book.id !== action.id);
  
      default:
        return state;
    }
  };
  ```

Reply:
Resolved